packages/renderer/src/lib/image/ImageColumnActions.spec.ts

```diff
@@ -40,6 +40,7 @@ const image: ImageInfoUI = {
   status: 'UNUSED',
   icon: undefined,
   badges: [],
+  digest: 'sha256:1234567890',
```

Comment:
Curious why digest is added here when looks optional (and optional handled)? Would prefer not add it here (and other tests) if not required.

Reply:
You are right, not needed!